android/modules/ui/src/java/ti/modules/titanium/ui/widget/tableview/TiTableViewRowProxyItem.java

```diff
@@ -552,6 +552,10 @@ protected void onMeasure(int widthMeasureSpec, int heightMeasureSpec)
 				} else {
 					h = Math.max(minRowHeight, height.getAsPixels(this));
 				}
+				// Make sure the height is greater than 1 (not 0 since image views default to 1)
```

Comment:
Removing these lines was the fix for "not growing beyond original height" error.
Now they are back and the error too.

Once again, this is the only place in the entire SDK where used assignment to LayoutParams.